apps/src/code-studio/components/libraries/LibraryListItem.jsx

```diff
@@ -94,7 +94,6 @@ export default class LibraryListItem extends React.Component {
           <button
             type="button"
             onClick={() => this.props.onRemove(library.name)}
-            style={styles.deleteButton}
```

Comment:
Why don't we need this style anymore?

Reply:
It wasn't being used (it doesn't even exist in the styles object)